addons/website/views/res_config_settings_views.xml

```diff
@@ -363,7 +363,7 @@
                                     </div>
                                 </div>
                             </div>
-                            <div class="col-12 col-lg-6 o_setting_box" id="robots_setting">
+                            <div class="col-12 col-lg-6 o_setting_box d-none" id="robots_setting">
```

Comment:
In master, we will remove the code, we just hide it in stable to prevent breaking customizations.